src/cim/components/ClusterDeployment/LocationsSelector.tsx

```diff
@@ -16,15 +16,12 @@ const LocationsLabelIcon: React.FC = () => {
         <>
           {t('ai:Select one or multiple locations to choose the hosts from.')}
           <br />
-          {t('ai:Keep the field empty to match <b>any</b> location.')}
+          {t('Keep the field empty to match <b>any</b> location.')}
           <br />
-          {
-            (t(
-              "ai:Set {{AGENT_LOCATION_LABEL_KEY}} label in Agent resource to specify it's location.",
-            ),
-            { AGENT_LOCATION_LABEL_KEY })
-          }
-          '
+          {t(
+            "Set <b>{{agent_location_label_key}}</b> label in Agent resource to specify it's location.",
+            { agent_location_label_key: AGENT_LOCATION_LABEL_KEY },
+          )}
```

Comment:
since we are using html tag here, should we use `Trans` instead ? IIRC `t` does not handle it

Reply:
I've changed it